internal/resources/fabric/service_token/models.go

```diff
@@ -176,14 +180,15 @@ func buildUpdateRequest(d *schema.ResourceData) []fabricv4.ServiceTokenChangeOpe
 	}
 
 	if !areSlicesEqual(oldZsideBandwidth, newZsideBandwidth) {
-		patches = append(patches, fabricv4.ServiceTokenChangeOperation{
-			Op:    "replace",
-			Path:  "/connection/supportedBandwidths",
-			Value: newZsideBandwidth,
-		})
+		patches = append(patches, []fabricv4.ServiceTokenChangeOperation{
+			{
```

Comment:
Unnecessary indentation. You can move the opening bracket up a line and remove the indent of the lines below it.